README.md

```diff
@@ -19,3 +19,30 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+-----
+
+A "good pivot" is a pivot that is placed in the middle range of an array.  According to the lecture slide on the intuition of the average case of quicksort, the probability that the randomly selected element is an element in the middle portion of the array is 1/2.
+
+When considering the median-to-three method for finding a pivot, I wrote out all permutations of three elements (e1, e2, and e3, where e1 is the first element, e2 is the middle element, and e3 is the last element).  I created a table, which looks like:
+
+| e1 | e2 | e3 |
+| -- | -- | -- |
+| 0  | 0  | 0  |
+| 0  | 0  | 1  |
+| 0  | 1  | 0  |
+| 1  | 0  | 0  |
+| 1  | 1  | 0  |
+| 0  | 1  | 1  |
+| 1  | 0  | 1  |
+| 1  | 1  | 1  |
+
+1 represents when an element is the median element, and 0 represents when an element is not the median element.  Since a good pivot is an element in the middle range of the array, e2 should be the median element if a good pivot was chosen.  From this fact and by looking at the table, e2 is the median element 4/8 = 1/2 = 50% of the time.  This means the median-to-three method is equally as likely to pick a good pivot compared to the average case method as shown in the lecture slides.
```

Comment:
Going in the right direction, but it still doesn't make sense that there's no difference in probabilities. How many different combinations of elements are there, what are the probabilities, and which ones result in a good pivot?